DuckDuckGo/Onboarding.swift

```diff
@@ -72,19 +79,51 @@ extension MainViewController {
 
 extension MainViewController: OnboardingDelegate {
     
+    static var onboardingSettingsPixelFired = false
+    static var onboardingExplorePrivacyPixelFire = false
```

Comment:
Initially I haven't seen that this is all included in Onboarding.swift - hence some thoughts about it:

I don't have strong opinion about it, but I'd consider moving all this "presentation/fire pixel" logic to OnboardingVC (or some Onboarding-related logic class) directly, as this seems to me like a logic that could be encapsulated within such class. This way MainVC will have less Onboarding logic. :)

If you think there is any merit to that, then few noes from me:
1. I know Settings screen needs to be bound to MainVC, but we probably can just do this through delegate (e.g. `onboarding(_ VC, willDisplaySettings: SettingsVC)` 
2. Since Onboarding is living only at the start, we probably can replace static vars with lazy ones (being a poor man's version of `dispatch_once`):
```
    private var fireSettingOpenedOnce: ()-> Void = {
        Pixel.fire(pixel: "mo_s")
        return {}
    }()
```
And just invoke `fireSettingOpenedOnce()`

Reply:
Great feedback.   The lazy function call there is 👌  but I've gone for a couple of instance vars in the VC instead as I think it's a bit clearer?